### ISSUE TITLE
High Contrast Themes Missing Selected Marking for Trees

### ISSUE THREAD

**Issue by colin-grant-work** [eclipse-theia/theia#10111]
### Bug Description:

In high-contrast themes, there is no marking of the selected item in trees:

VSCode (note orange border in tree and on tab):

![image](https://user-images.githubusercontent.com/62660806/133464953-5e8a82fe-e920-4d80-9597-60eb9ce97fa8.png)

Theia (note absence of marking in both tree and tab):

![image](https://user-images.githubusercontent.com/62660806/133465105-76a5e733-1ff2-4c2b-b92b-c0f30e2f4a40.png)

### Steps to Reproduce:

1. Open application
2. Set theme to high-contrast (either one)
3. Select an item in a tree.
4. Note that there is no indication of selection

### Additional Information

Areas where marking is missing:
 - [ ] Trees
 - [ ] Tab bars
 - [ ] Quick picks
 (Feel free to point out more, if you notice any)

- Operating System: RHEL7 (Electron)
- Theia Version: 05f289b3401fb32c0d3a8218ad9d33ef2758cc53

**Comment by vince-fugnitto** [comment]
@colin-grant-work should we close https://github.com/eclipse-theia/theia/issues/6743 in favor of this issue?

**Comment by colin-grant-work** [comment]
It looks like https://github.com/eclipse-theia/theia/issues/7353 has about the same amount of detail that I have, so maybe we close this one and #6743 in favor of that?

**Comment by vince-fugnitto** [comment]
> It looks like #7353 has about the same amount of detail that I have, so maybe we close this one and #6743 in favor of that?

Up to you :) Let's keep one open for sure.

**Comment by colin-grant-work** [comment]
Closing in favor of #6743, which has several more missing high-contrast theme areas in its comments.
